docs/src/dev-guide/components-core/index.md

```
@@ -15,9 +15,56 @@
 To build, we require some source dependencies, packages from package managers, and libraries built
 from source.
 
+### Build Environment
+
+A handful of packages and libraries are required to build CLP. There are two options to use them:
+
+* Install them on your machine and build CLP natively
+* Build CLP within a prebuilt docker container that contains the libraries;
+  However, this won't work if you need additional libraries that aren't already in the container.
+* If you just want to quickly build a CLP binary compatible with most platforms, you can run the following script:
+  ```shell
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)